packages/compose/src/hooks/use-preferred-color-scheme/index.ios.js

```diff
@@ -1,18 +1,13 @@
 /**
  * External dependencies
  */
-import { useDarkModeContext, eventEmitter } from 'react-native-dark-mode';
-
-// Conditional needed to pass UI Tests on CI
-if ( eventEmitter.setMaxListeners ) {
-	eventEmitter.setMaxListeners( 150 );
-}
+import { useColorScheme } from 'react-native';
 
 /**
  * Returns the color scheme value when it changes. Possible values: [ 'light', 'dark' ]
  *
  * @return {string} return current color scheme.
  */
-const usePreferredColorScheme = useDarkModeContext;
+const usePreferredColorScheme = useColorScheme;
 
-export default usePreferredColorScheme;
+export default usePreferredColorScheme;
```

Comment:
There seems to be a missing empty line at the end of the file.

Reply:
Fixed. Thanks!